src/passes/StackCheck.cpp

```diff
@@ -130,9 +119,19 @@ struct StackCheck : public Pass {
       return;
     }
 
-    // Pick appropriate names.
-    auto stackBaseName = Names::getValidGlobalName(*module, "__stack_base");
-    auto stackLimitName = Names::getValidGlobalName(*module, "__stack_limit");
+    // __stack_base and __stack_end are created by the toolchain (emscripten)
+    // If we don't find these globals, skip this pass.
```

Comment:
Skipping seems risky. How about erroring?

Reply:
Sounds good, I'll change that.